src/Service/CallService.php

```diff
@@ -583,17 +583,28 @@ public function decodeResponse(
             return [];
         }
 
-        $this->callLogger->debug('Response content: '.$responseBody);
+        // This if is statement prevents binary code from being used a string.
```

Comment:
typo in comment here
"from being used a string"

Reply:
Easeir to discuss this in a meeting with the other points you noted on this pull request